src/lib/Zikula/Bundle/CoreBundle/Twig/Extension/GettextExtension.php

```diff
@@ -50,73 +63,79 @@ public function getName()
     /**
      * @see __()
      */
-    public function __(\Twig_Environment $env, $message, $domain = null)
+    public function __(\Twig_Environment $env, $message, $domain = null, $locale = null)
```

Comment:
Current domain is already known to translator it is set to bundledomain in controller constructor. You can 'ask for different domain' using domain parameter. I need to configure cs fixer I hope to finish it today.